### DISCUSSION TITLE
Perfect component

### DISCUSSION THREAD

**Discussion by fcakyon** [SiddhantSadangi/st_login_form#10] [General]
@SiddhantSadangi @TheAhmedRmdan I just wanted to thank you for maintaining this great st component. I was looking for a similar component but there isn't any 👌🏻

**Comment by SiddhantSadangi** [comment]
Thanks for your kind words @fcakyon , glad you find this useful! :)

Also, thank you for Streamlit Image Comparison! I use it in another app of mine: https://imageworkdesk.streamlit.app/

**Reply by fcakyon** [reply]
Wow I'm glad you are making use of one of my components 🚀 

Keep the good work up 💯
